locale/shine/extensions/basecommands/frFR.json

```diff
@@ -0,0 +1,73 @@
+{
+	"CATEGORY": "Commandes de base",
+	"EJECT": "Éjecter",
+	"EJECT_TIP": "Éjecte le joueur de la command station/hive.",
+	"KICK": "Expulser",
+	"KICK_TIP": "Expulse le joueur du serveur.",
+	"KICK_NO_REASON": "Aucune raison",
+	"KICK_TROLLING": "Trolling",
+	"KICK_LANGUAGE": "Langage injurieux",
+	"KICK_MIC_SPAM": "Spamming mic",
+	"GAG": "Museler",
+	"GAG_TIP": "Empêche le joueur d'utiliser le chat écrit et vocal.",
+	"GAG_UNTIL_MAP_CHANGE": "Jusqu'au change map",
+	"UNGAG": "Démuseler",
+	"UNGAG_TIP": "Permet à un joueur muselé précédemment de s'exprimer à nouveau.",
+	"FORCE_RANDOM": "Forcer l'aléatoire",
+	"FORCE_RANDOM_TIP": "Déplace le(s) joueur(s) sélectionné(s) dans une équipe au hasard.",
+	"READY_ROOM": "Ready Room",
+	"READY_ROOM_TIP": "Déplace le(s) joueur(s) sélectionné(s) dans la ready room.",
+	"SET_TEAM": "Définir Équipe",
+	"SET_TEAM_TIP": "Déplace le(s) joueur(s) sélectionné(s) dans l'équipe choisie.",
+	"CHANGE_MAP": "Change map",
+	"CHANGE_MAP_TIP": "Change la carte immédiatement.",
+	"CALL_VOTE": "Demander Vote Map",
+	"CALL_VOTE_TIP": "Demande un vote map.",
+	"PLUGIN": "Plugin",
+	"STATE": "Statut",
+	"LOAD_PLUGIN": "Activer Plugin",
+	"UNLOAD_PLUGIN": "Désactiver Plugin",
+	"UNLOAD_PLUGIN_TIP": "Désactive temporairement le plugin.",
+	"UNLOAD_PLUGIN_SAVE_TIP": "Maintient le plugin désactivé.",
+	"LOAD_PLUGIN_TIP": "Active temporairement le plugin.",
+	"LOAD_PLUGIN_SAVE_TIP": "Maintient le plugin activé.",
+	"RELOAD_PLUGIN": "Recharger Plugin",
+	"ALLTALK_ENABLED": "Le all talk est activé.",
+	"ALLTALK_DISABLED": "Le all talk est désactivé.",
+	"CHEATS_TOGGLED": "{enabled:BoolToPhrase:activé|désactivé} cheats.",
+	"ALL_TALK_TAG": "[AllTalk]",
+	"ALLTALK_TOGGLED": "{enabled:BoolToPhrase:activé|désactivé} all talk.",
+	"ALLTALK_PREGAME_TOGGLED": "{enabled:BoolToPhrase:activé|désactivé} all talk pre-game.",
+	"ALLTALK_LOCAL_TOGGLED": "{enabled:BoolToPhrase:activé|désactivé} local all talk",
```

Comment:
Just noticed that these have lower-cased `enabled` when it should be `Enabled`, otherwise it won't work.